govcd/extension_test.go

```diff
@@ -24,7 +26,129 @@ func (vcd *TestVCD) Test_GetExternalNetwork(check *C) {
 	check.Assert(err, IsNil)
 	LogExternalNetwork(*externalNetwork)
 	check.Assert(externalNetwork.HREF, Not(Equals), "")
-	expectedType := "application/vnd.vmware.admin.extension.network+xml"
 	check.Assert(externalNetwork.Name, Equals, networkName)
-	check.Assert(externalNetwork.Type, Equals, expectedType)
+	check.Assert(externalNetwork.Type, Equals, types.MimeExtensionNetwork)
+}
+
+func (vcd *TestVCD) Test_CreateExternalNetwork(check *C) {
```

Comment:
For test organization (using tags) this function should be in `externalnetwork_test.go`

Reply:
Well from that perspective you are right, but from functionality point it is in right place as we match unitest file name against functions which resides there.

Reply:
In that case `extension_test.go` needs the tag `extnetwork`.